testbook/__init__.py

```diff
@@ -1,2 +1,2 @@
-from ._version import version as __version__
-from .testbook import testbook
+from testbook._version import version as __version__
```

Comment:
We shouldn't change from relative imports -- this causes problems otherwise with imports in certain situations it can be used in.

Reply:
Reverted